src/ontology/edits/oeo-shared.omn

```diff
@@ -2106,25 +2180,33 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/517",
 Class: OEO_00140006
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Private transport is a transport process in which people use their own vehicle for movement e.g. bicycle, motorcycle and cars, or walking.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A private transport is a passenger transport in which people use their own vehicle for movement e.g. bicycle, motorcycle and cars, or walking.",
```

Comment:
"... or walking"
The def of passenger transport clearly refers to a vehicle, though.

Reply:
I would be fine with excluding walking here.
@Vera-IER, what do you think?

Reply:
me too, walking is not something we model in our transport sector